src/Clock.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  currentTime: string,
+};
+
+const getUTCTime = () => new Date().toUTCString().slice(-12, -4);
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    currentTime: getUTCTime(),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
```

Comment:
It's better to wrap the window.setInterval callback in a separate method, e.g. 'updateTime', to improve code readability.